images-list

```diff
@@ -177,6 +179,7 @@ grafana/grafana rancher/mirrored-grafana-grafana 7.1.5
 grafana/grafana rancher/mirrored-grafana-grafana 7.4.5
 grafana/grafana rancher/mirrored-grafana-grafana 7.5.8
 grafana/grafana rancher/mirrored-grafana-grafana 7.5.11
+grafana/grafana rancher/mirrored-grafana-grafana 9.1.5
```

Comment:
I was under the impression that newer versions of grafana were not acceptable due to the change to AGPL licensing?

Reply:
We got approval from legal back in May. I am working on getting an official message here.